tasks/setup-Debian.yml

```diff
@@ -1,4 +1,20 @@
 ---
+- name: Ensure security packages are installed.
+  apt:
+    name:
+      - gnupg
+    state: present
+
+- name: Install postgres gpg key
+  apt_key:
+    url: "https://www.postgresql.org/media/keys/ACCC4CF8.asc"
+
+- name: Enable postgres repo
+  apt_repository:
+    repo: "deb http://apt.postgresql.org/pub/repos/apt/ {{ ansible_distribution_release }}-pgdg main"
```

Comment:
This is adding more than just version pinning—this PR seems to be doing a couple things (adding repo, changing the way the postgres_version variable works...).